org.eclipse.xtend.core.tests/src/org/eclipse/xtend/core/tests/validation/OverrideValidationTest.java

```diff
@@ -514,15 +514,24 @@ public void testClassMustBeAbstract_06() throws Exception {
 	}
 
 	/**
-	 * Two incompatible exceptions; the marker is on the first one
+	 * Three incompatible exceptions from different supertypes;
+	 * the marker is set on the offending exception
 	 */
 	@Test public void testIncompatibleThrowsClause_06() throws Exception {
-		var source = "class Foo extends test.ExceptionThrowing { override nullPointerException() throws java.io.IOException, java.io.FileNotFoundException {} }";
+		var source = "class Foo extends test.ExceptionThrowing implements test.ExceptionThrowingInterface, test.ExceptionThrowingInterface2 {"
+				+ "override nullPointerException() throws NullPointerException, java.io.IOException, java.io.FileNotFoundException {} "
+				+ "}";
 		XtendClass xtendClass = clazz(source);
+		var expectedSuffix = " is not compatible with throws clause in "
```

Comment:
Maybe and a tiny `the` like this: `is not compatible with the throws clause in`

Reply:
OK, done!